pkgs/by-name/lu/luci-go/package.nix

```
@@ -0,0 +1,45 @@
+{
+  buildGoModule,
+  fetchgit,
+  lib,
+}:
+let
+  commit = "500493c154652d6986a34b341e98df244ae1ad0d";
+in
+buildGoModule rec {
+  pname = "luci-go";
+  version = "20241031";
```

```suggestion
  version = "0-unstable-2024-10-31";
```

If not a tag or a real version

---

Ok, changed.